controllers/topics.controller.js

```diff
@@ -0,0 +1,7 @@
+const { selectAllTopics } = require("../models/topics.model")
+
+exports.getAllTopics = (req, res) => {
+    selectAllTopics().then((topics) => {
+        res.status(200).send({topics})
+    })
```

Comment:
Do you think a catch block would be useful here 🤔

Reply:
yes, it is 
I will add it